packages/styles/scss/components/masthead/_masthead-leftnav.scss

```diff
@@ -137,11 +142,32 @@
         }
 
         .#{$prefix}--masthead__side-nav--submemu-title {
-          @include carbon--type-style(expressive-heading-03, true);
+          @include use-carbon-productive-tokens();
+          @include carbon--type-style(productive-heading-02);
 
+          display: flex;
           color: $text-01;
-          padding: $carbon--spacing-05;
+          padding: 0 $carbon--spacing-05;
+          height: 3rem;
```

Comment:
Minor thing, I wonder if we should use `$container-03` here?

cc: @annawen1 @asudoh

Reply:
@jeffchew Nice, I didn't know about this token, thanks!